imports/ui/pages/resources/resourcesSingle.jsx

```diff
@@ -209,6 +208,56 @@ export class ResourcesSingle_default extends React.Component{
     }
 }
 
+class ExternalApps extends React.Component{
+    renderLinks(applications) {
+
+        const resourceName = _.get(this.props.resource, 'searchableData.name', this.props.selfLink);
+        const resourceKind = this.props.resource.searchableData.kind;
+
+        const filteredApps = applications.filter( (app) => {
+            const nameRegEx = new RegExp(app.nameMatch, 'i');
+            const kindRegEx = new RegExp(app.kindMatch, 'i');
+            return nameRegEx.test(resourceName) && kindRegEx.test(resourceKind);
+        });
+        
+        const urlTemplate = {
+            'kind': resourceKind,
+            'name': resourceName
+        };
+
+        if(filteredApps && filteredApps.length > 0 ) {
+            return (
+                <ul className="externalApp list-unstyled">
+                    {
+                        filteredApps.map( (app) => {
+                            let output = Mustache.render(app.url, urlTemplate);
+                            return <li key={app.name}><a href={output} target="_blank">{app.name} <i className="fa fa-external-link"></i></a></li>
+                        })
+                    }
+                </ul>
+            );
+        } else {
+            const orgLink = `/${FlowRouter.getParam('baseOrgName')}/org`;
```

Comment:
`pathFor('org')` should work here

Reply:
are we able to use the pathFor helper in react components?